runner/internal/shim/api/server.go

```diff
@@ -12,6 +12,7 @@ import (
 type TaskRunner interface {
 	Run(context.Context, shim.DockerImageConfig) error
 	GetState() shim.RunnerStatus
+	SubmitStop(bool)
```

Comment:
Why is it named `SubmitStop` and not simply `Stop`?

Reply:
Fixed.